src/main/java/edu/ucla/library/bucketeer/verticles/LargeImageVerticle.java

```diff
@@ -1,22 +1,56 @@
 
 package edu.ucla.library.bucketeer.verticles;
 
+import java.io.File;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+
+import javax.naming.ConfigurationException;
+
+import com.nike.moirai.ConfigFeatureFlagChecker;
+import com.nike.moirai.FeatureFlagChecker;
+import com.nike.moirai.Suppliers;
+import com.nike.moirai.resource.FileResourceLoaders;
+import com.nike.moirai.typesafeconfig.TypesafeConfigDecider;
+import com.nike.moirai.typesafeconfig.TypesafeConfigReader;
+
 import info.freelibrary.util.Logger;
 import info.freelibrary.util.LoggerFactory;
+import info.freelibrary.util.StringUtils;
 
 import edu.ucla.library.bucketeer.Config;
 import edu.ucla.library.bucketeer.Constants;
+import edu.ucla.library.bucketeer.Features;
+import edu.ucla.library.bucketeer.HTTP;
 import edu.ucla.library.bucketeer.MessageCodes;
+import edu.ucla.library.bucketeer.Op;
 
+import io.vertx.core.buffer.Buffer;
 import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.HttpResponse;
 import io.vertx.ext.web.client.WebClient;
 
+/**
+ * A verticle that handles the processing of large images. We have a limit on the size of images that AWS Lambda can
+ * handle, so we can use this to pass the image conversion off to another instance of Bucketeer that's set up to handle
+ * large image conversion and upload.
+ */
 public class LargeImageVerticle extends AbstractBucketeerVerticle {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LargeImageVerticle.class, Constants.MESSAGES);
 
+    private static final String LARGE_IMAGE_ENDPOINT = "/images/{}/{}"; // image-id and file-path
```

Comment:
I think it would improve clarity to match the name to the operation that the endpoint maps to (`loadImage`).